maintainers/maintainer-list.nix

```diff
@@ -10526,4 +10526,10 @@
     github = "jbcrail";
     githubId = 6038;
   };
+  higebu = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add higebu`` and sort it alphabetically.